plugins/webapi/tools/value/utils.go

```diff
@@ -0,0 +1,15 @@
+package value
```

Comment:
>    * The spammer itself can be a new plugin, but the webapi (handler + route reg) part should stay in the `webapi` package. So the `spammer.Spammer` instance is in the new plugin, and you can control it from the `webapi`. I think it is better, because it also signals that you can control it from other modules too, not just webapi. (or what would be the motivation to have it as a new plugin?)

Spammer is more like a standalone plugin or tool for users to play around IMO, and the webapi is only available once the spammer is enabled. So putting them all in the `spammer` plugin makes it clearer and easier to use, similar to `dapps/networkdelay`, and `spammer` should probably be moved to `pluginmgr/core` instead.